src/Microsoft.VisualStudio.Editors/PropPages/WPF/ApplicationPropPageVBWPF.vb

```diff
@@ -930,7 +930,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages.WPF
             End Function
 
             Public Overrides Function GetHashCode() As Integer
-                Throw New NotImplementedException()
+                Return -1937169414 + EqualityComparer(Of String).Default.GetHashCode(Value.ToLowerInvariant)
```

Comment:
I think you want `StringComparer.OrdinalIgnoreCase.GetHashCode(Value)`  My understanding is that `OrdinalIgnoreCase` and `ToLowerInvariant` are different

Reply:
yeah, thats the correct fix.